spec/models/user_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+describe User do
+  let(:user){ build(:user) }
+
+  # Happy
+  it "is valid with default attributes" do
+    expect(user).to be_valid
+  end
+
+  it "has password length bigger then 4" do
```

Comment:
I think you may have these two mixed up; this test if the password is smaller than 4, but says bigger.